client/decs.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-conversation-form';
```

Comment:
What's the point of this one-line file?

Reply:
This declare was to fix the fact that react-conversational-form is a naturally a typescript language so we have to declare the module in a separate file so we can tell the compiler you don't have to check for typescript